### PR TITLE
Add typography styles

### PR DESCRIPTION
**Pull request by klzns**
Add typography styles tokens `.t-heading-1`, `.t-heading-2`, `.t-heading-3`, `.t-heading-4`, `.t-heading-5`, `.t-heading-6`, `.t-body`, `.t-small`, `.t-mini`, `.t-action`, `.t-action--small`, `.t-action--large` and `.t-code`. All of those classes have their responsive counterparts, i.e., `.t-body-s`, `.t-body-ns`, `.t-body-m`, `.t-body-l` and `.t-body-xl`.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,6 @@
 node_modules
 npm-debug.log
 .DS_STORE
-config.json
\ No newline at end of file
+config.json
+package-lock.json
```
Comment (lbebber) [inline .gitignore:5]: 🤙

File: config.js [hunk 2]
```diff
@@ -271,7 +271,100 @@ module.exports = {
   "maxWidths": [1, 2, 4, 8, 16, 32, 48, 64, 96],
   "heights": [1, 2, 4, 8, 16],
   "typography":{
-    "measure": [30, 34, 20]
+    "measure": [30, 34, 20],
+    "styles": {
+      "heading-1": {
+        "fontFamily": "Fabriga, -apple-system, BlinkMacSystemFont, avenir next, avenir, helvetica neue, helvetica, ubuntu, roboto, noto, segoe ui, arial, sans-serif",
+        "fontWeight": "700",
+        "fontSize": "48px",
```
Comment (lbebber) [inline config.js]: nao vai ser definida com rem mais?

**Review by lbebber — APPROVED** [review]

File: config.js (hunk 2 above)
Comment (klzns) [reply config.js]: Creio que não, estou a mando do @gabrielgalc

**Comment by gabrielgalc** [comment]
Boa! Não tinha nem me ligado!

**Pull request merged**